src/features/user/Profile/components/_SettingsModal.tsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
```

Comment:
Is this component being used any more? It seems to be referred to only in `src/features/user/Profile/index.tsx`, and the relevant code is commented out.

Reply:
Removed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1467/commits/e4d76e258a2fd11e0299bf9cbebdd0644d286256